torch_geometric/sampler/base.py

```diff
@@ -0,0 +1,62 @@
+from abc import ABC, abstractmethod
+from typing import Dict, List, NamedTuple, Tuple, Union
+
+import torch
+
+from torch_geometric.data import Data, HeteroData
+from torch_geometric.data.feature_store import FeatureStore
+from torch_geometric.data.graph_store import GraphStore
+from torch_geometric.typing import EdgeType, NodeType
+
+# An input to a sampler a tensor of node indices:
+SamplerInput = torch.Tensor
```

Comment:
nitpick: Do we really need this, why not just use torch.Tensor directly.

Reply:
Soon, this will be split into `NodeSamplerInput` and `EdgeSamplerInput`, so I prefer to leave it as-is in the current PR.